test/llvm-intrinsics/memmove.ll

```diff
@@ -60,6 +73,18 @@
 ; CHECK-LLVM: [[tmp4:%[0-9]+]] = bitcast %struct.SomeStruct* [[local]] to [[type]]
 ; CHECK-LLVM: call void @llvm.lifetime.end.p0i8({{i[0-9]+}} {{-?[0-9]+}}, [[type]] [[tmp4]])
 
+; CHECK-LLVM-LABEL: @test_array
+; CHECK-LLVM: %[[#ALLOCA:]] = alloca [16 x i8], align 1
+; CHECK-LLVM: %[[#TMP0:]] = bitcast [16 x i8]* %[[#ALLOCA]] to i8*
+; CHECK-LLVM: call void @llvm.lifetime.start.p0i8(i64 -1, i8* %[[#TMP0]])
+; CHECK-LLVM: %[[#TMP1:]] = bitcast [16 x i8]* %[[#ALLOCA]] to i8*
+; CHECK-LLVM: call void @llvm.memcpy.p0i8.p1i8.i32(i8* %[[#TMP1]], i8 addrspace(1)* %out, i32 128, i1 false)
```

Comment:
Sorry, but I still don't understand how we can copy 128 bytes into array of 16 bytes.  @MrSidims, am I missing something?